media-gfx/freecad/freecad-0.16.ebuild

```diff
@@ -0,0 +1,143 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit cmake-utils eutils fortran-2 multilib python-single-r1 fdo-mime
```

Comment:
I don't see the need for `eutils.eclass` after purging `epatch` and `prune_libtool_files` and `multilib.eclass` anymore